readme.md

```diff
@@ -106,8 +114,7 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
-
+// Your code starts here... I'm not sure how to do this without looking it up. I'll be reviewing Express this week (and the entire MEAN stack) which is what I have by far the most trouble with.
 ```
 
```

Comment:
Let me know if you'd like any help with this after reviewing this week. That being said, it's totally OK to look things up on this quiz!